### PR TITLE
Update spm quickstarts with CI script

### PR DESCRIPTION
**Pull request by jrcrespoh**
- [x] abtesting
- [x] crashlytics
- [x] database
- [x] performance

### CODE REVIEW

File: scripts/test_spm.sh
Comment (morganchen12) [inline scripts/test_spm.sh:72]: nit: remove this if it's unused

File: .github/workflows/crashlytics.yml
Comment (morganchen12) [inline .github/workflows/crashlytics.yml:105]: This is much nicer than the previous CI script, great work!

**Review by morganchen12 — APPROVED** [review]
LGTM, but please get Paul's approval before merging.

File: scripts/test_spm.sh
Comment (paulb777) [inline scripts/test_spm.sh:26]: Please investigate a generalized test for any Xcode >= 12.5. I'd prefer for this not to start breaking with 13.1 etc.

Comment (paulb777) [reply scripts/test_spm.sh:26]: Something like https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh#L97 could help

**Review by paulb777 — COMMENTED** [review]
This is a great improvement for maintainability. Thanks!

One thing to solve and then LGTM

File: scripts/test_spm.sh
Comment (jrcrespoh) [reply scripts/test_spm.sh:72]: Done ✅

File: scripts/test_spm.sh
Comment (jrcrespoh) [reply scripts/test_spm.sh:26]: Done ✅

**Review by paulb777 — APPROVED** [review]

**Review by granluo — APPROVED** [review]

**Pull request merged**
